### ISSUE TITLE
Timeout at a connection configuration level

### ISSUE THREAD

**Issue by profsmallpine** [jakob/Postico#605]
I would love to see a timeout feature. For example, I could configure a connection to timeout after 5 minutes of not using it. This would be an added protection for prod databases.

**Comment by profsmallpine** [comment]
Of course, I’m sorry if this is not inline with the product, but it would sure be nice to have. I have 4 connections for each environment of the product I am working on. At the end of the day, I might think I’m in my local database when I’m actually not. Creating the ability to force a timeout would just decrease the likeliness that I would accidentally make an update to the wrong database.

**Comment by pamann** [comment]
Would really like this too, for the same reason as above. I have multiple environments and want to prevent accidentally leaving a production database open. I tried turning off the "keep connections alive" setting and adding an SSH timeout on my bastion instance, but Postico still seems to be automatically reconnecting (switching tables gives me a "connection unexpectedly closed" error but I can write to the same table that I left open without any warning). Ideally I'd have to close and reopen the server if it times out.
